go-controller/pkg/ovn/service.go

```diff
@@ -112,7 +112,7 @@ func (ovn *Controller) syncServices(services []interface{}) {
 	// 'nodeportServices'.
 	gateways, stderr, err := ovn.getOvnGateways()
 	if err != nil {
-		klog.Errorf("failed to get ovn gateways. Not syncing nodeport"+
+		klog.Errorf("Failed to get ovn gateways. Not syncing nodeport"+
```

Comment:
Not your issue, but since you are changing this line we need a space at the end -- s/nodeport/nodeport /g